bindings/rust/evmc-sys/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "evmc-sys"
+version = "6.2.0-dev"
+authors = ["Alex Beregszaszi <alex@rtfs.hu>"]
+license = "Apache-2.0"
+repository = "https://github.com/ethereum/evmc"
+description = "Bindings to EVMC (low-level)"
+categories = ["external-ffi-bindings"]
+edition = "2018"
```

Comment:
2019

Reply:
This is a Rust specific field. 2018 is valid, 2019 isn't.